### PR TITLE
Introduce a sig-compute charter

### PR DESCRIPTION
**Pull request by iholder101**
**What this PR does / why we need it**:
Introduces a sig-compute charter.
See https://github.com/kubevirt/community/blob/main/sig-TEMPLATE/README.md for more info.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: sig-compute/charter.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- node-labeller
+- and more
+
+#### Virtualization
+- virt-launcher
+- libvirt / QEMU configuration
+- CPU features
+- Dedicated CPUs
+- Hypervisor features (e.g. HyperV)
+- and more
```
Comment (vladikr) [inline sig-compute/charter.md:39]: I think you should also explicitly mention the devices management: generic host devices interface, GPU and vGPU discovery configuration.

**Comment by fabiand** [comment]
@iholder101 @xpivarc how can we converge this?

**Comment by xpivarc** [comment]
> @iholder101 @xpivarc how can we converge this?

I would suggest a review from approver.

**Comment by iholder101** [comment]
Can you please have another look?
@jean-edouard @xpivarc @vladikr

**Comment by iholder101** [comment]
> > @iholder101 @xpivarc how can we converge this?
> 
> I would suggest a review from approver.

Anything missing from your end?
If not I'd appreciate LGTMing :)

**Review by jean-edouard — APPROVED** [review]

**Comment by vladikr** [comment]
/approve
looks good me .

**Pull request merged**